toybox.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: toybox
+  version: "0.8.12"
+  epoch: 0
+  description: "A BSD-licensed alternative to busybox"
+  copyright:
+    - license: BSD-0-Clause
```

Comment:
Is this correct and spdx organized license? https://github.com/landley/toybox?tab=0BSD-1-ov-file#readme 

also is this license OSI approved?

Reply:
> spdx organized license

https://spdx.org/licenses/0BSD.html

Yes, it is. I just need to change the `BSD-0-Clause` into a proper short identifier `0BSD`